### PR TITLE
Format profile

### PR DESCRIPTION
**Pull request by kathrynljackson**
### Participating Group Members:
* KJ
 
### Code Highlights:
 * Reformat `Profile`
* The `display` style property of `EditProfile` can now be toggled on a button click 

### Have Tests Been Added?
- [x] No.
- [ ] Yes, but all tests are not passing. 
- [ ] Yes, and all are passing.
 
### Any background context you want to provide?
* n/a
 
### Message/Questions for reviewer:
* n/a
 
### Issues: 
* Addresses #20 #29 #18
 
### Screenshots (if appropriate): 
Screen Shot 2020-12-12 at 6 09 05 PM

Screen Shot 2020-12-12 at 6 31 43 PM

 
### Tracking Consistency:
- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

**Review by aemcdonald — APPROVED** [review]
Awesome job, I really like the toggle!

File: code/web/src/modules/user/EditProfile.js [hunk 1]
```diff
@@ -32,14 +34,21 @@ class EditProfile extends Component {
     }
 
     showEditInfo = () => {
-        //this is the function that will show and hide the menu
+        if (this.state.shown === true) {
+            this.setState({ shown: false })
+        } else {
+            this.setState({ shown: true }) 
+        }
+
```
Comment (kathrynljackson) [inline code/web/src/modules/user/EditProfile.js:42]: The edit menu is shown based on the state of h component.

File: code/web/src/modules/user/Profile.js [hunk 2]
```diff
@@ -38,20 +38,22 @@ const Profile = (props) => (
     <Grid>
       <GridCell style={{ padding: '2em', textAlign: 'center' }}>
         {/*<Tile style={{}} image={icon}></Tile>*/}
-        <ImageTile width={300} height={530} shadow={level1} image={`${ APP_URL }/images/stock/men/1.jpg`} />
+        <ImageTile style={{ borderRadius: '5px'}} object-fit='cover' width={300} height={300} shadow={level1} image={`${ APP_URL }/images/stock/men/1.jpg`} />
```
Comment (kathrynljackson) [inline code/web/src/modules/user/Profile.js:41]: The size of the user image is now limited.

**Review by stacyp2006 — APPROVED** [review]
Thanks for cleaning up those comments.

File: code/web/src/modules/user/EditProfile.js (hunk 1 above)
Comment (nathanielmillard) [reply code/web/src/modules/user/EditProfile.js:42]: Nice, this is looking good!

File: code/web/src/modules/user/Profile.js (hunk 2 above)
Comment (nathanielmillard) [reply code/web/src/modules/user/Profile.js:41]: Glad to see this worked in!

**Review by nathanielmillard — APPROVED** [review]
I love how much these few small changes can really shape things up!

**Pull request merged**